packages/runtime/src/state/modules/api-resources.ts

```diff
@@ -48,11 +63,12 @@ export function getSerializedState(state: State): SerializedState {
   }
 
   Array.from(state.entries()).forEach(([resourceType, resources]) => {
-    const particularResourceMap: { id: string; value: any }[] = []
+    const particularResourceMap: { id: string; value: any; locale?: any }[] = []
```

Comment:
Unintended `any` here?

Reply:
No, this is intentional. Conditional typing of `locale` in `SerializedState` (`APIResourceLocale<key>`) makes TS to reject anything but `any` here or forces us to add a type cast to `resourceMap[resourceType] = particularResourceMap` in line 75 below. I opted in for `any` as the lesser of two evils.